### PR TITLE
Graphql and Apollo

### PR DESCRIPTION
**Pull request by BernabeFelix**
Setup basic Graphql and Apollo.

Customer Admin is connected and also home, let's leave it like this and integrate apollo as needed.

### CODE REVIEW

File: client/src/components/CustomerAdmin/AdminPromotion.jsx [hunk 1]
```diff
@@ -34,37 +35,52 @@ class AdminPromotion extends Component {
...
+                    padding={$padding}
+                  />
+                }
+                rightIconButton={
+                  <IconButton
+                    touch
+                    onClick={this.deletePromo}
+                    tooltip="Eliminar"
+                  >
+                    <DeleteIcon color="#ee3335" />
```
Comment (paulsancer) [inline client/src/components/CustomerAdmin/AdminPromotion.jsx]: You've already imported the red color, so: `<DeleteIcon color={$red} />`

**Review by paulsancer — APPROVED** [review]
Just a silly change, please do before merge.

I hope you can tell me more about this later today.

**Review by paulsancer — CHANGES_REQUESTED** [review]
`localhost:8080` renders nothing:

```
Uncaught TypeError: Cannot read property 'promotions' of undefined
    at PromotionsResults._react2.default.createElement (webpack:///./src/components/Home/PromotionsResults.jsx?:45)
```

**Comment by BernabeFelix** [comment]
@Polkien did you run `npm install` both backend and frontend?

**Review by paulsancer — APPROVED** [review]

**Pull request merged**